### PR TITLE
patching create node issue within the utils.py

### PR DESCRIPTION
**Pull request by nh916**
Discovered this issue with the new API on staging and develop servers. API gives back a "can_edit" filed within the JSON, but that is not used for python SDK or the Excel Uploader but is used in the Web SDK for criptapp

### CODE REVIEW

**Review by ericmuckley — APPROVED** [review]

**Review by bearmit — APPROVED** [review]

**Comment by nh916** [comment]
@InnocentBug just glad we got another thing crossed off!

I noticed some checks were failing, is that okay or is there something that I need to do before merging in?

**Comment by InnocentBug** [comment]
should be fine to merge now.
But at some point we should take a look.
My comment was more meant to mean, that you didn't introduce this failing with this PR.

**Review by InnocentBug — APPROVED** [review]

**Pull request merged**
